src/json/amp-carousel_with_amp-list.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Can you please use some of our own hosted images, for example: `/json/related_products.json` should work quite well in this case.

Reply:
I will edit the api to point here and the mustache template to reference it properly